.github/workflows/backend-test.yml

```diff
@@ -11,33 +11,21 @@ on:
   pull_request:
     branches:
       - 'main'
+
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+
 jobs:
   backend:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         java-version: [ 17 ]
-    # Service containers to run with `runner-job`
-    services:
-      packit-db:
-        image: mrcide/packit-db:${{ github.head_ref || github.ref_name }}
-        # Set health checks to wait until postgres has started and add container name
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-          --name packit-db
-        ports:
-          - 5432:5432
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-      - name: Connect to PostgreSQL
-        run: db/scripts/add-schema-and-dummy
-        env:
-          POSTGRES_HOST: localhost
-          POSTGRES_PORT: 5432
+      - name: Run dependencies
+        run: ./scripts/run-dependencies
```

Comment:
consolidated our approach to running up dependencies - this script can be used for local dev and for testing